resharper/resharper-unity/src/Unity/Core/Application/UI/Help/ShowUnityHelp.cs

```diff
@@ -87,6 +87,11 @@ private FileSystemPath GetDocumentationRoot()
             var appPath = mySolutionsManager.Solution?.GetComponent<UnityVersion>().GetActualAppPathForSolution();
             var contentsPath = UnityInstallationFinder.GetApplicationContentsPath(appPath);
             var root = contentsPath.Combine("Documentation");
+
+            // I see /home/ivan-shakhov/Unity/Hub/Editor/2021.2.4f1/Editor/Data/Documentation/Documentation/en path on my machine
+            if (root.Combine("Documentation").ExistsDirectory)
+                root = root.Combine("Documentation");
```

Comment:
Very minor nitpick - let's pull `root.Combine()` out into a variable, so we don't have to parse and create the path twice.